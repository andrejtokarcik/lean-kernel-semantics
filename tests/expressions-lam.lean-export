1 #NS 0 A
2 #NS 1 a
1 #US 0
1 #ES 1
2 #EV 0
3 #EL #BD 2 2 2
4 #EL #BI 1 1 3
