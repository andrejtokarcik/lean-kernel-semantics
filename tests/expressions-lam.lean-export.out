<T>
  <k>
    .
  </k>
  <names>
    0 |-> *empty*
    1 |-> A
    2 |-> A.a
  </names>
  <universes>
    0 |-> 0
    1 |-> 1
  </universes>
  <expressions>
    1 |-> 1
    2 |-> #0
    3 |-> λ (A.a : #0) → #0
    4 |-> λ {A : 1} → λ (A.a : #0) → #0
  </expressions>
  <def-types>
    .Map
  </def-types>
  <def-values>
    .Map
  </def-values>
  <exit-code>
    0
  </exit-code>
</T>
